### DISCUSSION TITLE
Network Tab next to Servers / Workstations / Mixed to Manually Add Network Assets

### DISCUSSION THREAD

**Discussion by dkleckner498** [amidaware/tacticalrmm#1035] [Ideas]
Tried checking on this specific Idea and did not see anything.  Adding a Network Tab and being able to add Network Assets Manually however you can choose within that specific Location / Group a Server to be the associated check and add a check on that server to ping an IP address which would be the network item (Firewall, Switch, AP and other) which would give a green check if its online and would report the associated Network asset as being pingable / online.  Something simple to report but captures the network asset as well as some type of check without needing to do SNMP right now.

**Comment by dinger1986** [comment]
You can ping devices on the network, manually add them

**Comment by silversword411** [comment]
This sounds like a "Network Discovery" feature request...can probably do some scripted workarounds till there's time for a big feature ;)

**Comment by dkleckner498** [comment]
Yes, haha I was trying to go around the Network Discovery as I know that's probably a much bigger undertaking to try to implement and was in hopes of something maybe easier?  However, unfortunately I am no developer so all of the development (I understand to a certain degree) can be foreign.  Grateful for the developers and their hard work.

**Reply by dinger1986** [reply]
My plan with the linux agent was look into snmp connectors and collectors on linux, looked into it with windows and was a nightmare. The new script (it's on discord but not GitHub yet) I wrote would in theory collect a list of all devices on the network and could be saved.

**Reply by dkleckner498** [reply]
Ok ya that might work as an agent based discovery using Linux.  However to associate and export from the agent asset to a new Tab for that location using something like a Network Tab.  Ultimately for network discovery you would end up specifying which IP Subnets to have the agent scan for devices and then enter the OID associated to the SNMP output (which in itself can be a nightmare)
